services/movie_session.py

```diff
@@ -24,6 +24,17 @@ def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
     return MovieSession.objects.get(id=movie_session_id)
 
 
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    session = MovieSession.objects.get(id=movie_session_id)
+    tickets = Ticket.objects.filter(movie_session=session)
```

Comment:
```suggestion
    tickets = Ticket.objects.filter(movie_session_id=movie_session_id)
```
No need to do two queries